codebuild-terraform-apply/code_build_terraform.tf

```diff
@@ -0,0 +1,58 @@
+resource "aws_codebuild_project" "code_pipeline_terraform" {
+  name        = "code-pipeline-terraform"
+  description = "Run terraform validate and then terraform apply"
+
+  service_role = data.aws_iam_role.execution_role.arn
+
+  artifacts {
+    type = "CODEPIPELINE"
+  }
+
+  cache {
+    type  = "LOCAL"
+    modes = ["LOCAL_DOCKER_LAYER_CACHE", "LOCAL_SOURCE_CACHE"]
+  }
+
+  environment {
+    compute_type                = "BUILD_GENERAL1_SMALL"
+    image                       = "gdscyber/cyber-security-concourse-base-image:latest"
```

Comment:
I wonder if we should make this a var with our image as the default

Reply:
That should definitely be the case, I'll go ahead and make that change now